src/python_testing/matter_testing_support.py

```diff
@@ -154,11 +156,32 @@ def parse_pics(lines=typing.List[str]) -> dict[str, bool]:
     return pics
 
 
-def read_pics_from_file(filename: str) -> dict[str, bool]:
-    """ Reads a dictionary of PICS from a file. """
-    with open(filename, 'r') as f:
-        lines = f.readlines()
-        return parse_pics(lines)
+def parse_pics_xml(contents: str) -> dict[str, bool]:
+    pics = {}
+    mytree = ET.fromstring(contents)
+    for pi in mytree.iter('picsItem'):
+        name = pi.find('itemNumber').text
+        support = pi.find('support').text
+        pics[name] = int(json.loads(support.lower())) == 1
+    return pics
+
+
+def read_pics_from_file(path: str) -> dict[str, bool]:
+    """ Reads a dictionary of PICS from a file (ci format) or directory (xml format). """
+    if os.path.isdir(os.path.abspath(path)):
+        pics_dict = {}
+        for filename in glob.glob(f'{path}/*.xml'):
+            with open(filename, 'r') as f:
+                contents = f.read()
+                pics_dict.update(parse_pics_xml(contents))
+        print(pics_dict)
+        return pics_dict
+
+    else:
+        print("notdir")
```

Comment:
same here - is this debug code?

Reply:
yup - now you've seen my shameful hacks :) Removed.